javascript/chronometer.js

```diff
@@ -1,34 +1,41 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => this.currentTime++, 1000)
   }
```

Comment:
Great job, Thu!
Here don't forget to write the condition for the callback function!